### PR TITLE
Bump to sdk 0.10.0

### PR DESCRIPTION
**Pull request by galak**

### CODE REVIEW

**Review by nashif — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]
Tested 0.10.0 with a couple of basic samples

File: cmake/toolchain/zephyr/host-tools.cmake [hunk 1]
```diff
@@ -15,7 +15,7 @@ if(NOT ZEPHYR_SDK_INSTALL_DIR)
   return()
 endif()
 
-set(REQUIRED_SDK_VER 0.9.5)
+set(REQUIRED_SDK_VER 0.10.0)
```
Comment (nashif) [inline cmake/toolchain/zephyr/host-tools.cmake]: actually, lets give people more time with this, otherwise it will break downstream CI

**Review by nashif — DISMISSED** [review]

**Review by aescolar — DISMISSED** [review]
Shouldn't this PR update the documentation to point to it?
https://docs.zephyrproject.org/latest/getting_started/installation_linux.html?highlight=0.9.5#install-the-zephyr-software-development-kit-sdk 
~~I guess, it would also need to be linked from https://www.zephyrproject.org/developers/#downloads?~~

**Comment by erwango** [comment]
> Shouldn't this PR update the documentation to point to it?

Saw that also (#14258)

**Pull request closed without merging**